lib/ci-helper.ts

```diff
@@ -521,7 +521,16 @@ GitGitGadget needs an email address to Cc: you on your contribution, so that you
      * @param commentID the ID of the PR comment to handle
      */
     public async handleComment(repositoryOwner: string, commentID: number): Promise<void> {
-        const comment = await this.github.getPRComment(repositoryOwner, commentID);
+        let comment: IPRComment;
+
+        try {
+            comment = await this.github.getPRComment(repositoryOwner, commentID);
+        } catch (e) {
+            const error = e as Error;
```

Comment:
I wonder whether we can ensure that this exception was thrown because the comment was not found, i.e. a 404?

We would still want the run to fail upon, say, 500...

Reply:
Judging by [this code](https://github.com/gitgitgadget/gitgitgadget/blob/403cc505b4749d866bbccdc472754b078215661a/tests/github-glue.test.ts#L526), I believe we should be able to filter by this via `if (e.status === 404) [...]`.

Reply:
> Judging by [this code](https://github.com/gitgitgadget/gitgitgadget/blob/403cc505b4749d866bbccdc472754b078215661a/tests/github-glue.test.ts#L526), I believe we should be able to filter by this via `if (e.status === 404) [...]`.

Updated to check the status and check it is a `RequestError`.  I chose to export `RequestError` from `github-glue.ts` to hide octokit from `ci-helper.ts`.  If you prefer a direct import I can change it (back).

Tested by running the code as part of `github-glue.test.ts` with a bad comment number.

Reply:
This is excellent!